From 368f6ff4fe04f14e73730e40a50752eda1af7f14 Mon Sep 17 00:00:00 2001
From: "haolong.zhang" <haolong.zhang@ck-telecom.com>
Date: Mon, 11 May 2015 11:13:21 +0800
Subject: [PATCH] [Feature] TP _Gesture Fuc to user space

    [id]:87846
    [TYPE]
       FEATURE
    [Products]
       hicam_baseline
    [Requirement]
       N/A
    [Solution Description]
       N/A
    [Other info]
    Useage:
    1: attr: /sys/class/ms-touchscreen-msg20xx/device/gesture_wakeup_mode
    2: set attr 0 to disable _Gesture Fuc
    3: set below args to enable varis _Gesture Fuc
     GESTURE_WAKEUP_MODE_DOUBLE_CLICK_FLAG     0x0001    //0000 0000
     GESTURE_WAKEUP_MODE_UP_DIRECT_FLAG        0x0002    //0000 0000
     GESTURE_WAKEUP_MODE_DOWN_DIRECT_FLAG      0x0004    //0000 0000
     GESTURE_WAKEUP_MODE_LEFT_DIRECT_FLAG      0x0008    //0000 0000
     GESTURE_WAKEUP_MODE_RIGHT_DIRECT_FLAG     0x0010    //0000 0000
     GESTURE_WAKEUP_MODE_m_CHARACTER_FLAG      0x0020    //0000 0000
     GESTURE_WAKEUP_MODE_W_CHARACTER_FLAG      0x0040    //0000 0000
     GESTURE_WAKEUP_MODE_C_CHARACTER_FLAG      0x0080    //0000 0000
     GESTURE_WAKEUP_MODE_e_CHARACTER_FLAG      0x0100    //0000 0001
     GESTURE_WAKEUP_MODE_V_CHARACTER_FLAG      0x0200    //0000 0010
     GESTURE_WAKEUP_MODE_O_CHARACTER_FLAG      0x0400    //0000 0100
     GESTURE_WAKEUP_MODE_S_CHARACTER_FLAG      0x0800    //0000 1000
     GESTURE_WAKEUP_MODE_Z_CHARACTER_FLAG      0x1000    //0001 0000

Change-Id: Iba8dccbe5cfefc1b3ff28c0fe63bb3c30d83068e
---
 .../touchpanel/msg2133_and_ektf2k_v3/mstar_drv_platform_interface.c     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mediatek/custom/common/kernel/touchpanel/msg2133_and_ektf2k_v3/mstar_drv_platform_interface.c b/mediatek/custom/common/kernel/touchpanel/msg2133_and_ektf2k_v3/mstar_drv_platform_interface.c
index 97048ac..e1ebff1 100644
--- a/mediatek/custom/common/kernel/touchpanel/msg2133_and_ektf2k_v3/mstar_drv_platform_interface.c
+++ b/mediatek/custom/common/kernel/touchpanel/msg2133_and_ektf2k_v3/mstar_drv_platform_interface.c
@@ -58,7 +58,7 @@ void MsDrvInterfaceTouchDeviceSuspend(struct early_suspend *pSuspend)
     DBG("*** %s() ***\n", __func__);
 
 #ifdef CONFIG_ENABLE_GESTURE_WAKEUP
-    g_GestureWakeupMode = 0x0001; // Enable all gesture wakeup mode for testing
+    //g_GestureWakeupMode = 0x1FFF; // Enable all gesture wakeup mode for testing
 
     if (g_GestureWakeupMode != 0x0000)
     {
-- 
1.9.1

